src/nft/pages/asset/Asset.tsx

```diff
@@ -51,35 +49,17 @@ const Asset = () => {
         properties={{ collection_address: contractAddress, token_id: tokenId }}
         shouldLogImpression
       >
-        {asset && collection ? (
+        {!!asset && !!collection && (
```

Comment:
hm i /think/ this may be problematic, if asset===true and collection===false, wont' this expression resolves to `false`, in some cases writing the string `false` to the dom?

Reply:
casting these params to Bools prevents the React compiler from writing them to the DOM. This can be tested on the preview by editting the collection address or tokenId to invalid values